### PR TITLE
Add load balancing implementation for Apache in documentation

### PR DESCRIPTION
**Pull request by TheoMathurin**
Addresses #12152 

Reorganize section on load balancing to include configuration for Apache, remove trailing slash in Apache proxy config.

### CODE REVIEW

**Comment by bryevdv** [comment]
Thanks for the PR @TheoMathurin ! It looks great to me at a glance but I've pinged @tcmetzger for a docs review when he is able to look at it!

**Comment by TheoMathurin** [comment]
@bryevdv 

Alright! The instructions are really minimal as there isn't much more to say, but I hope this helps.

File: sphinx/source/docs/user_guide/server.rst [hunk 1]
```diff
@@ -1183,10 +1183,22 @@ distributed among individual servers.
     The Bokeh server is horizontally scalable. To add more capacity, you
     can run more servers behind a load balancer.
 
-Nginx can help with load balancing. This section describes some of the basics
-of one possible configuration, but please also refer to the
-`Nginx load balancer documentation`_. For instance, there are different
-strategies available for choosing what server to connect to next.
+You can run as many Bokeh servers as you need. In the following, let us
+suppose that you have three Bokeh servers running on separate ports:
```
Comment (tcmetzger) [inline sphinx/source/docs/user_guide/server.rst:1187]: Following https://docs.bokeh.org/en/latest/docs/dev_guide/documentation.html#principles-of-good-style, I'd replace the first person 'us' with something else:

```suggestion
You can run as many Bokeh servers as you need. The following examples
are based on a setup with three Bokeh servers running on three different
ports:
```

File: sphinx/source/docs/user_guide/server.rst [hunk 2]
```diff
@@ -1183,10 +1183,22 @@ distributed among individual servers.
...
+.. code-block:: sh
+
+    bokeh serve myapp.py --port 5100
+    bokeh serve myapp.py --port 5101
+    bokeh serve myapp.py --port 5102
+
+The sections below propose basic configurations, but please also refer to the
+`Nginx load balancer documentation`_ or the `Apache proxy balancer module
+documentation`_. For instance, there are different strategies available for
+choosing how incoming connections are distributed among instances.
```
Comment (tcmetzger) [inline sphinx/source/docs/user_guide/server.rst:1198]: For clarity, I'd turn the first sentence into two sentences:
```suggestion
The sections below propose basic configurations based on this setup. See the
`Nginx load balancer documentation`_ or the `Apache proxy balancer module
documentation`_ for more detailed information. For instance, there are
different strategies available to define how incoming connections are
distributed among server instances.
```

File: sphinx/source/docs/user_guide/server.rst [hunk 3]
```diff
@@ -1239,6 +1239,38 @@ code below uses ``proxy_pass http://myapp;``.
 
     }
 
+Apache
+''''''
+
+First make sure you have enabled the ``proxy_balancer`` and ``rewrite``
```
Comment (tcmetzger) [inline sphinx/source/docs/user_guide/server.rst:1245]:
```suggestion
First, make sure you have enabled the ``proxy_balancer`` and ``rewrite``
```

File: sphinx/source/docs/user_guide/server.rst [hunk 4]
```diff
@@ -1239,6 +1239,38 @@ code below uses ``proxy_pass http://myapp;``.
...
+modules. Then you need to add balancers for both http and websocket protocols.
```
Comment (tcmetzger) [inline sphinx/source/docs/user_guide/server.rst:1246]:
```suggestion
modules. 

Add balancers for both http and websocket protocols:
```

File: sphinx/source/docs/user_guide/server.rst [hunk 5]
```diff
@@ -1239,6 +1239,38 @@ code below uses ``proxy_pass http://myapp;``.
...
+    </Proxy>
+
+    <Proxy "balancer://myapp_ws">
+        BalancerMember "ws://127.0.0.1:5100/myapp"
+        BalancerMember "ws://127.0.0.1:5101/myapp"
+        BalancerMember "ws://127.0.0.1:5102/myapp"
+        ProxySet lbmethod=byrequests
+    </Proxy>
+
+Finally you can proxy connections to the two balancers:
```
Comment (tcmetzger) [inline sphinx/source/docs/user_guide/server.rst:1264]:
```suggestion
Finally, you can proxy connections to the two balancers:
```

**Review by tcmetzger — COMMENTED** [review]
@TheoMathurin Thank you, this is a very welcome addition to the docs! I just added some minor stylistic suggestions, looking forward to having this information in the docs!

**Comment by TheoMathurin** [comment]
Thanks for the review @tcmetzger!

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks for the PR @TheoMathurin !

**Pull request merged**